production-marathon/marathon.json.b

```diff
@@ -1,43 +1,40 @@
 {
-	"id": "caf",
-	"groups": [{
-		"id": "audit",
-		"apps": [{
-			"id": "audit-service",
-			"cpus": 0.25,
-			"mem": 768,
-			"instances": 1,
-			"container": {
-				"docker": {
-					"image": "${DOCKER_REGISTRY}/audit-service:3.1.0",
-					"network": "BRIDGE",
-					"portMappings": [{
-						"containerPort": 8080,
-						"hostPort": 0,
-						"protocol": "tcp",
-						"servicePort": ${CAF_AUDIT_SERVICE_PORT}
-					}],
-					"forcePullImage": true
-				},
-				"type": "DOCKER"
-			},
-			"env": {
-				"_JAVA_OPTIONS": "-Xms384m -Xmx384m",
-				"CAF_ELASTIC_HOST_AND_PORT": "${CAF_ELASTIC_HOST_AND_PORT}",
-				"CAF_ELASTIC_CLUSTER_NAME": "${CAF_ELASTIC_CLUSTER_NAME}",
-				"CAF_ELASTIC_NUMBER_OF_REPLICAS": "1",
-				"CAF_ELASTIC_NUMBER_OF_SHARDS": "5",
-				"CAF_LOG_LEVEL": "INFO"
-			},
-			"healthChecks": [{
-				"gracePeriodSeconds": 300,
-				"intervalSeconds": 60,
-				"maxConsecutiveFailures": 3,
-				"path": "/",
-				"portIndex": 0,
-				"protocol": "HTTP",
-				"timeoutSeconds": 20
-			}]
-		}]
-	}]
+    "id": "audit-prod",
```

Comment:
I see the id has changed here. Is this consistent with other services deployed in Mesos/Marathon e.g. job service. Have we decided on a consistent naming standard for parent group and service name across all these services?